alacritty_terminal/src/term/mod.rs

```diff
@@ -1875,8 +1891,24 @@ impl<T: EventListener> Handler for Term<T> {
                     let template = Cell { bg: template.bg, ..Cell::default() };
                     self.grid.clear_viewport(template);
                 }
+
+                let last_visible_line_index = num_lines - 1;
```

Comment:
You can't inline those, since `clippy` bugs.

Reply:
It's not a bug. It's just that you probably should implement the `intersects_range` method properly.